Microsoft.Toolkit.Uwp/Helpers/CameraHelper/CameraHelper.cs

```diff
@@ -34,12 +33,18 @@ public class CameraHelper : IDisposable
         private MediaFrameReader _frameReader;
         private MediaFrameSourceGroup _group;
         private MediaFrameSource _frameSource;
+        private List<MediaFrameFormat> _frameFormatsAvailable;
 
         /// <summary>
         /// Gets the currently selected <see cref="MediaFrameSource"/>MediaFrameSource
         /// </summary>
         public MediaFrameSource FrameSource { get => _frameSource; }
 
+        /// <summary>
+        /// Gets the available MediaFrameFormats on this sources
+        /// </summary>
+        public List<MediaFrameFormat> FrameFormatsAvailable { get => _frameFormatsAvailable; }
```

Comment:
Would you be adding a setter so that the user can select the format for preview?

Reply:
no since there is a getter on the FrameSource, I would defer to the API already available on that object